decidim-budgets/app/views/decidim/budgets/admin/projects/update_category.js.erb

```diff
@@ -0,0 +1,26 @@
+<% if flash.now[:error].present? %>
```

Comment:
Can we somehow merge the code in `update_category.js.erb`, `update_scope.js.erb` and `update_selected.erb`?

It seems very similar code in all these files with only few changes in each. I would try to find a way to make this code the same for all of these cases.